examples/power_devices.py

```diff
@@ -40,22 +40,80 @@
 
 oneview_client = OneViewClient(config)
 
-# Get power devices utilization with defaults
-print("Get power devices utilization")
+power_device_information = {
+    "name": "MyPdd",
+    "ratedCapacity": 40
+}
+
+power_device_information_ipdu = {
+    "hostname": "172.18.8.12",
+    "username": "",
+    "password": ""
```

Comment:
Since the IP, username and password is sensitive data, I believe it should be placed in the config file.